packages/widget/src/components/address-input/address-input.ts

```diff
@@ -76,7 +78,7 @@ export class AddressInput extends BaseComponent {
             }
           }}
           @input=${(evt: Event) =>
-            this.handleAddressChange((evt.target as HTMLTextAreaElement).value)}
```

Comment:
Are you sure that it shouldn't be a textAreaElement?

Reply:
solved